src/main/java/imageEnergiesAndSeams/Image.java

```diff
@@ -49,37 +51,56 @@ public int[][] removeHorizontal(int[] seamToRemove) {
         int height = currentPic.length;
         int width = currentPic[0].length;
         int[][] newImage = new int[height - 1][width];
+        BufferedImage newBufferedImage = new BufferedImage(width,height - 1, BufferedImage.TYPE_INT_RGB);
         for (int x = 0; x < width; x++) {
             for (int y = 0; y < height-1; y++) {
                 if (y < seamToRemove[x]){
+                    newBufferedImage.setRGB(x, y, currBufferedImage.getRGB(x, y));
                     newImage[y][x] = currentPic[y][x];
                 }
                 else{
                     newImage[y][x] = currentPic[y+1][x];
+                    newBufferedImage.setRGB(x, y, currBufferedImage.getRGB(x, y + 1));
                 }
             }
         }
+        currBufferedImage = newBufferedImage;
         return newImage;
     }
 
     public int[][] removeVertical(int[] seamToRemove) {
         int height = currentPic.length;
         int width = currentPic[0].length;
         int[][] newImage = new int[height][width - 1];
+        BufferedImage newBufferedImage = new BufferedImage(width - 1,height, BufferedImage.TYPE_INT_RGB);
```

Comment:
It's a waste to create a buffered image every time you remove one seam. You only care about creating an image once you remove all the seams